jsdoc/tag-defs/access.js

```diff
@@ -1,13 +1,16 @@
-module.exports = function(accessTagTransform, trimWhitespaceTransform) {
-  var name = 'access'
-  accessTagTransform.addTag(name);
-  accessTagTransform.addValue('private');
-  accessTagTransform.addValue('protected');
-  accessTagTransform.addValue('public');
-
+module.exports = function(createDocMessage) {
+  STANDARD_VALUES = ['private', 'protected', 'public'];
   return {
-    name: name,
-    docProperty: 'access',
-    transforms: [trimWhitespaceTransform, accessTagTransform]
+    name: 'access',
+    transforms: function(doc, tag, value) {
+      var value = value.trim();
+      if (STANDARD_VALUES.indexOf(value) !== -1) {
```

Comment:
This does not restrict the value of access at all.
The only thing is that standard values are also applied directly to the doc as a boolean property (as well as being assigned to the `access` property.

So a doc containing `@access private` will end up with:

```
{ access: 'private', private: true }
```

while a doc with `@access custom` will look like:

```
{ access: 'custom' }
```